documents/src/ecc/usage.md

```diff
@@ -1,76 +1,63 @@
-# Usage
-
-## Usage
+### Usage
+```sh
+ecc [OPTIONS] <SOURCE_PATH> [EXPORT_EVENT_HEADER]
+```
+  
+Compiles and generates a bpf object from the provided SOURCE_PATH path for the specified eBPF program.
 
-The only file you will need to write is:
+### example
```

Comment:
Add more examples:

1. A minimal example to generate a json config and an object, with only a source file
2. A more complex example with source file and a header file
3. Generate btf hub tar